tests/IceRpc.Conformance.Tests/Transports/MultiplexedListenerConformanceTests.cs

```diff
@@ -109,43 +109,20 @@ public async Task Connect_fails_if_listener_is_disposed()
         IListener<IMultiplexedConnection> listener = provider.GetRequiredService<IListener<IMultiplexedConnection>>();
         var clientTransport = provider.GetRequiredService<IMultiplexedClientTransport>();
 
-        Task connectTask = ConnectAsync(clientTransport);
+        await using IMultiplexedConnection connection = clientTransport.CreateConnection(
```

Comment:
Should we make a successful connection first to make sure the ConnectAsync doesn't fail for some other reason?

Reply:
Yes, I reworked this test